### PR TITLE
Copy dist files to root + add package/bower tasks

### PR DESCRIPTION
**Pull request by jledentu**
* As said in the #193 discussion, we need to copy the dist files to the root dir, in order to ensure the compatibility until the 1.0 version.

* This PR also adds two other Gulp tasks: `package` (to copy the samples in `dist` and generate the release zip archive) and `bower` (to generate the `bower.json`file), as in **Chart.js** and **chartjs-plugin-datalabels**.

* The `travis.yml` has been updated to enable automatic release and deployment.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,25 +1,35 @@
...
+# https://github.com/travis-ci/travis-ci/issues/5538#issuecomment-225025939
 
-addons:
-  firefox: latest
-  apt:
-    sources:
-      - google-chrome
-    packages:
-      - google-chrome-stable
+deploy:
```
[the remark this answers is not in the document]
Comment (jledentu) [reply .travis.yml]: @simonbrunel I added a `MAINTAINING.md` file as you suggested. 👍

I can see that "_npm autoupdate_" is configured on **cdnjs** for `Chart.js`. Are you able to configure that for this lib (currently _git autoupdate_)?

[the remark this answers is not in the document]
Comment (simonbrunel) [reply .travis.yml]: You can submit a PR similar to cdnjs/cdnjs#8401 but for the [chartjs-plugin-zoom package](/cdnjs/cdnjs/tree/master/ajax/libs/chartjs-plugin-zoom).

[the remark this answers is not in the document]
Comment (jledentu) [reply .travis.yml]: Ok! I submitted this PR: https://github.com/cdnjs/cdnjs/pull/13269

File: scripts/release.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```
Comment (simonbrunel) [inline scripts/release.sh]: Did you make sure that this file have the [executable (+x) git flag](https://git-scm.com/docs/git-update-index#git-update-index---chmod-x)?

Comment (jledentu) [reply scripts/release.sh]: Yes I did!

**Comment by jledentu** [comment]
@benmccann Do you think this PR is ready to be merged?

**Comment by benmccann** [comment]
Yep, looks good to me. I wasn't sure if there was anything else we needed Simon to do first, but I'll go ahead and merge it now. Thanks!

**Pull request merged**